messages/main/es.json

```diff
@@ -9,7 +9,7 @@
   "menu-file": "Archivo",
   "menu-change-language": "Cambiar idioma...",
   "menu-change-language-title": "Ingrese el idioma (e.g., 'en' or 'es')",
-  "menu-import-tiles-file": "Archivo (.asar, .tar)",
+  "menu-import-tiles-file": "File (.tar)",
```

Comment:
do you know if this was on purpose @gmaclennan ?

Reply:
No, from my side this was not on purpose. What I've realised is that these files need to be managed by CrowdIn - if we change the translations in Github then Crowdin does not update and then it overwrites with a previous translation. I got caught by that on Mapeo Mobile a few times. I've updated the string now in Crowdin, should come through here.